docs/products/compute/compute-instances/guides/metadata/index.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Overview of the Metadata Service"
+description: "Learn how to automate server provisioning on the Linode platform through the new Metadata service and cloud-init."
+keywords: ["user data", "metadata", "cloud-init", "cloudinit"]
+published: 2023-07-24
+modified_by:
+  name: Linode
+authors: ["Linode"]
+_build:
+  list: false
+noindex: true
+---
+
+{{< content "metadata-beta-notice" >}}
+
+When deploying Compute Instances, it's almost always necessary to perform additional configuration before you can host your website or run your workloads. This configuration might include creating a new user, adding an SSH key, or installing software. It could also include more complex tasks like configuring a web server or other software that runs on the instance. Performing these tasks manually can be tedious and is not ideal at larger scales. To automate this configuration, Linode offers two provisioning automation tools: Metadata (covered in this guide), and [StackScripts](/docs/products/tools/stackscripts/).
+
+## Overview
+
+Linode's Metadata service provides a convenient method to automate software configuration when deploying a Compute Instance. It is an API that's accessible only from within a provisioned Compute Instance and provides relevant metadata to that instance. The Metadata service is designed to be consumed by [cloud-init](https://cloudinit.readthedocs.io/en/latest/), an industry standard software that automates cloud instance initialization. This allows you to use the same tool across multiple cloud providers, enabling a pathway for provisioning your systems as part of a multi-cloud infrastructure strategy.
+
+The Metadata service provides both *instance data* and optional *user data*, both of which are explained below:
+
+-   **Instance data:** The instance data includes information about the Compute Instance, including its label, plan size, region, host identifier, and more.
+-   **User data:** User data is one of the most powerful features of the Metadata service and allows you to define your desired system configuration, including creating users, installing software, configuring settings, and more. User data is supplied by the user when deploying, rebuilding, or cloning a Compute Instance. This user data can be written as a cloud-config file, or it can be any script that can be executed on the target distribution image, such as a bash script.
+
+    User data can be submitted directly in the Cloud Manager, Linode CLI, or Linode API. It's also often programmatically provided through IaC (Infrastructure as Code) provisioning tools like [Terraform](/docs/guides/how-to-build-your-infrastructure-using-terraform-and-linode/).
+
+When a Compute Instance first boots up, cloud-init runs locally on the system, accesses the metadata, and then configures your system using that metadata.
+
+## Comparison to StackScripts
```

Comment:
This is outside the scope of this PR but it seems like it would be nice to have a comparison with things like Ansible too

Reply:
I'll also log this as feedback for docs supporting the GA release. I can see what the product team thinks about this.